libnd4j/include/ops/declarable/helpers/cpu/softmax.cpp

```diff
@@ -155,11 +155,14 @@ namespace sd {
                         T max = -DataTypeUtils::max<T>();
                         T sum(0.f);
 
+#ifndef __NEC__
 #pragma omp simd reduction(maxT:max)
+#endif
```

Comment:
That should probably become a macro like most of the other pragma stuff already is

Reply:
the same here. just re-write it with PRAGMA_* defined in include/system/openmp_pragmas.h

Reply:
@quickwritereader @treo same after chatting with @quickwritereader implemented the above